src/components/icons/IconStatus/IconStatus.tsx

```diff
@@ -0,0 +1,161 @@
+import * as React from 'react';
+
+import {
+  ExclamationCircleIcon,
+  InProgressIcon,
+  OffIcon,
+  PausedIcon,
+  SyncAltIcon,
+  UnknownIcon,
+} from '@patternfly/react-icons';
+import { global_danger_color_100 as dangerColor } from '@patternfly/react-tokens/dist/js/global_danger_color_100';
+
+import { faSpin } from './utils';
+
+type IconProps = {
+  title: string;
+  'data-test'?: string;
```

Comment:
replace all with `data-test-id`
this is console convention cc @metalice @yaacov